src/main/resources/templates/admin.html

```diff
@@ -40,9 +40,10 @@
     </li>
 </ul>
 
-<footer th:replace="fragments :: footer"></footer>
-
 </div>
 
+<div th:replace="fragments :: footer-spacer"></div>
+<footer th:replace="fragments :: footer"></footer>
+
```

Comment:
You'll see this in most of the files, where I added the footer-spacer fragment and the footer outside of the main content div.